libp2p/transport/transport_interface.py

```diff
@@ -1,22 +1,29 @@
 from abc import ABC, abstractmethod
 
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from multiaddr import Multiaddr
+    from libp2p.peer.id import ID
+    from libp2p.network.connection.raw_connection_interface import IRawConnection
+    from .listener_interface import IListener
+    from .typing import THandler
```

Comment:
This doesn't seem like it should need to be under the `TYPE_CHECKING` context block as that module doesn't import from anywhere else in the library.